network/address.go

```diff
@@ -88,6 +88,13 @@ const (
 	ScopeLinkLocal    Scope = "link-local"
 )
 
+var emptyAddress = Address{}
```

Comment:
As discussed, this might be a candidate to move inside the function, for tighter scope.